bento/tools/_decomposition.py

```diff
@@ -112,37 +112,32 @@ def decompose(
 def rmse(tensor, tensor_mu):
     return np.sqrt((tensor[tensor != 0] - tensor_mu[tensor != 0]) ** 2).mean()
 
-
-@track
 def to_tensor(
```

Comment:
i think we can take this out, superseded by `tools._colocation._colocation_tensor()`